Gems/Atom/Tools/MaterialCanvas/Code/Source/Document/MaterialGraphCompiler.cpp

```diff
@@ -141,7 +141,9 @@ namespace MaterialCanvas
             // safely generate the material SRG and material type that only contain variables referenced in the shaders. Without tracking
             // this, all variables would be included in the SRG and material type. The shader compiler would eliminate unused variables from
             // the compiled shader code. The material type would fail to build if it referenced any of the eliminated variables.
-            BuildMaterialSrgForCurrentNode();
+            // Note: The MaterialSrg has been replaced with the MaterialParameters - struct, which is generated automatically from the
+            // MaterialProperties with a shaderInput connection
+            // BuildMaterialSrgForCurrentNode();
```

Comment:
Do we still need the commented-out line in the future? If not please remove

Reply:
unclear. I've added a `deprecated` comment to that function. but i don't think we should remove it just yet.